sys/include/net/ng_pktdump.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
  * @brief   Priority of the pktdump thread
  */
 #ifndef NG_PKTDUMP_PRIO
-#define NG_PKTDUMP_PRIO                 (PRIORITY_MIN - 1)
+#define NG_PKTDUMP_PRIO                 (PRIORITY_MAIN - 1)
```

Comment:
Looks like. Didn't debug further as raising the priority helps and main() sits in shell.

The effect is that the pktdump loop never starts, thus no packets get printed. I think it even stalled the nomac thread.